hack/verify-no-vendor-cycles.sh

```diff
@@ -18,16 +18,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
 export GO111MODULE=auto
 
-staging_repos=($(ls "${KUBE_ROOT}/staging/src/k8s.io/"))
+staging_repos=()
+while IFS=$'\n' read -r line; do 
+  array+=("$line"); 
+done < <(ls "${KUBE_ROOT}/staging/src/k8s.io/")
```

Comment:
It seems that "array" is not used after that. Is "staging_repos" correct in this part?

Reply:
Thank you. Careless mistake